runtime/j9vm/exports.cmake

```diff
@@ -344,17 +344,14 @@ if(NOT JAVA_SPEC_VERSION LESS 15)
 	jvm_add_exports(jvm
 		JVM_RegisterLambdaProxyClassForArchiving
 		JVM_LookupLambdaProxyClassFromArchive
-		JVM_IsCDSDumpingEnabled
```

Comment:
My preference would be to keep the symbol in the group for the version in which it was introduced; see lines 404+ for an existing example. The result would put this before the `endif()` below.
```
	if(JAVA_SPEC_VERSION LESS 23)
		jvm_add_exports(jvm
			JVM_IsCDSDumpingEnabled
		)
	endif()
```

Reply:
Moved the methods in question back, and added `if(JAVA_SPEC_VERSION LESS 23)` within the original group.